code/game/objects/items/RCD.dm

```diff
@@ -944,38 +961,15 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 // Ranged RCD
 /obj/item/construction/rcd/arcd
 	name = "advanced rapid-construction-device (ARCD)"
-	desc = "A prototype RCD with ranged capability and extended capacity. Reload with iron, plasteel, glass or compressed matter cartridges."
-	max_matter = 300
-	matter = 300
+	desc = "A prototype RCD with ranged capability and infinite capacity."
+	max_matter = INFINITY
+	matter = INFINITY
```

Comment:
Why are we giving the ARCD infinite matter

Reply:
Because it's an admin tool just like the short ranged admin RCD.

https://github.com/tgstation/tgstation/blob/552262ba15fadc1e441b499f9c98025435630b6a/code/game/objects/items/RCD.dm#L937-L941

I mean it is right? Because the wiki page says so
![Screenshot_20230304-135349_Chrome](https://user-images.githubusercontent.com/110812394/222885579-5dc91cc1-71aa-4729-8dd9-b40b77b822cf.jpg)
In fact I think this should replace the short ranged admin RCD because obviously long range is better but I don't know if it's an non admin item then I'll revert the changes

Reply:
It also seems the linked cult group in the wiki has been removed from the code base
https://tgstation13.org/wiki/Clockwork_Cult
So I think it's safe to say this should be an admin only tool

Reply:
apparently runtime uses the ARCD while multiz.dmm uses the admin combat RCD. that's funny

Reply:
We will keep both rcds then.  I'll resolve this